panther_analysis_tool/analysis_utils.py

```diff
@@ -129,10 +150,61 @@ def add_classified_analysis(
             self.queries.append(classified_analysis)
 
 
+@dataclasses.dataclass
+class AnalysisFilters:
+    filters: Dict[str, List]
+    filters_inverted: Dict[str, List]
+
+    @staticmethod
+    def from_filter_list(filters: List[str]) -> "AnalysisFilters":
```

Comment:
This whole function was just copied from [parse_filters in main.py](https://github.com/panther-labs/panther_analysis_tool/pull/422/files#diff-4c3406e46ca13890fa64d295f86ed01e0b0bcd3d28031f1fc97978bb8fca2da3L2015) and I added a couple little changes

Reply:
aside from switching this up to be a class, the main logic looks to be the addition of new filter keys, which lgtm.

that said, unit testing this class and its instance methods might be nice.